nixos/tests/postgresql-wal-receiver.nix

```diff
@@ -0,0 +1,75 @@
+import ./make-test.nix ({ pkgs, ...}:
+
+let
+  postgresqlPackage = pkgs.postgresql_11;
```

Comment:
I've performed a test over PG 9.6 and 9.4

For 9.6 I had to add
```
        max_wal_senders = 10
        max_replication_slots = 10
```
to extraConfig for that to work. Maybe let's include unconditionally?

For 9.4 it didn't work at all:
```
LOG:  invalid value for parameter "wal_level": "replica"
HINT:  Available values: minimal, archive, hot_standby, logical.
```

Reply:
`max_wal_senders` and `max_replication_slots` added.

The `wal_level` option has been set to `archive` for backward compatibly, as it's an accepted alias for `replica` on newer versions.